src/Planet.php

```diff
@@ -0,0 +1,108 @@
+<?php
+
+namespace PlanetApp;
+
+class Planet
+{
+    protected int $id;
```

Comment:
is there a particular reason these are type hinted with just 'string' and the others with '?string' ?

Reply:
yep - just `string` means it will error if its anything other than string, `?string` means it can be null or string, this is useful as sometimes these fields dont have any data from the DB and this stops it throwing an error